cmd/containerd-nydus-grpc/pkg/command/flags.go

```diff
@@ -73,38 +71,35 @@ func buildFlags(args *Args) []cli.Flag {
 		},
 		&cli.StringFlag{
 			Name:        "daemon-mode",
-			Value:       DefaultDaemonMode,
-			Usage:       "set daemon working `MODE`, one of \"multiple\", \"shared\" or \"none\"",
+			Value:       defaultDaemonMode,
+			Usage:       "spawning nydusd daemon mode, legal values include \"multiple\", \"shared\" or \"none\"",
 			Destination: &args.DaemonMode,
 		},
 		&cli.StringFlag{
 			Name:        "fs-driver",
-			Value:       FsDriverFusedev,
-			Usage:       "backend `DRIVER` to serve the filesystem, one of \"fusedev\", \"fscache\"",
+			Value:       defaultFsDriver,
+			Usage:       "fulfill image service based on what fs driver, possible values include \"fusedev\", \"fscache\"",
 			Destination: &args.FsDriver,
 		},
 		&cli.StringFlag{
 			Name:        "log-level",
 			Value:       defaultLogLevel.String(),
-			Usage:       "set the logging `LEVEL` [trace, debug, info, warn, error, fatal, panic]",
+			Usage:       "logging level, possible values \"trace\", \"debug\", \"info\", \"warn\", \"error\"",
 			Destination: &args.LogLevel,
 		},
 		&cli.BoolFlag{
 			Name:        "log-to-stdout",
-			Usage:       "log messages to standard out rather than files.",
+			Usage:       "print log messages to STDOUT",
 			Destination: &args.LogToStdout,
 		},
 		&cli.StringFlag{
 			Name:        "nydus-image",
-			Value:       "",
-			Usage:       "set `PATH` to the nydus-image binary, default to lookup nydus-image in $PATH",
+			Usage:       "`nydus-image` binary path, it will be searched from $PATH if this option is not provided",
 			Destination: &args.NydusImagePath,
 		},
 		&cli.StringFlag{
 			Name:        "nydusd",
-			Value:       "",
-			Aliases:     []string{"nydusd-path"},
```

Comment:
Can we still keep the alias?

Reply:
Better not I suppose, thus to keep the consistency everywhere and simply the flows on how to use nydus